CMakeLists.txt

```diff
@@ -1,97 +1,25 @@
-cmake_minimum_required(VERSION 3.5)
-
+cmake_minimum_required(VERSION 3.10)
 project(mimick_vendor)
 
 find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_vendor_package REQUIRED)
 
-macro(build_mimick)
-
-  set(cmake_commands)
-  set(cmake_configure_args -Wno-dev)
-  list(APPEND cmake_configure_args -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_install)
-
-  if(WIN32)
-    if(DEFINED CMAKE_GENERATOR)
-      list(APPEND cmake_configure_args -G ${CMAKE_GENERATOR})
-    endif()
-    if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "^(x86_|x86-|AMD|amd|x)64$")
-      list(APPEND cmake_configure_args -A x64)
-    endif()
-  endif()
-
-  if(DEFINED CMAKE_BUILD_TYPE)
-    if(WIN32)
-      build_command(_build_command CONFIGURATION ${CMAKE_BUILD_TYPE})
-      list(APPEND cmake_commands "BUILD_COMMAND ${_build_command}")
```

Comment:
Does `ament_vendor()` take care of this too? I can't tell from the implementation 🤔

Reply:
It doesn't. I'm unable to reproduce the warning that brought on that change, and I haven't seen that pattern in any other vendor packages.

Reply:
Okay good to know 👍🏼